java/java.lsp.server/vscode/package.json

```diff
@@ -140,21 +140,6 @@
 				"title": "Compile Workspace",
 				"category": "Java"
 			},
-			{
-				"command": "java.generate.getters.menu",
```

Comment:
A lot of registrations is removed. However I assume they are somehow added back, right?

Reply:
CodeActions that have set `CodeActionKind.Source` automatically appear as commands under the `Source Action...` popup menu item in the VSCode client. There is no need for their manual registration.